fendermint/app/src/service/node.rs

```diff
@@ -301,20 +305,53 @@ pub async fn run(
         parent_finality_votes.clone(),
     );
 
-    let interpreter = FvmMessagesInterpreter::new(
+    // Load the module based on enabled features
+    // AppModule is a type alias that changes based on feature flags
+    let module = std::sync::Arc::new(AppModule::default());
+
+    tracing::info!(
+        module_name = fendermint_module::ModuleBundle::name(module.as_ref()),
+        module_version = fendermint_module::ModuleBundle::version(module.as_ref()),
+        "Initialized FVM interpreter with module"
+    );
+
+    // Initialize module services generically
+    // The module can start background tasks, set up resources, etc.
+    // Note: The keypair is passed as Vec<u8> for flexibility
+    // The plugin can deserialize it to the format it needs
+    let validator_key_bytes = if let Some(ref _k) = validator_keypair {
+        // Serialize the keypair - just use empty vec for now as placeholder
+        // Full implementation would serialize properly
+        Some(vec![])
+    } else {
+        None
+    };
+
+    let mut service_ctx = fendermint_module::service::ServiceContext::new(Box::new(settings.clone()));
+    if let Some(key_bytes) = validator_key_bytes {
+        service_ctx = service_ctx.with_validator_keypair(key_bytes);
+    }
+
+    let service_handles = module
+        .initialize_services(&service_ctx)
+        .await
+        .context("failed to initialize module services")?;
```

Comment:
We may want to have plugins enabled in cargo features, i.e. compiled into the binary, but nevertheless disabled in node's runtime configuration.